components/dashboard/src/user-settings/Preferences.tsx

```diff
@@ -142,6 +170,37 @@ export default function Preferences() {
                         </p>
                     </div>
                 </div>
+
+                <h3 className="mt-12">Timeout </h3>
+                <p className="text-base text-gray-500 dark:text-gray-400">Customize timeout setting for workspace.</p>
+                <div className="mt-4 max-w-xl">
+                    <h4>Workspace timeout</h4>
```

Comment:
**suggestion**: Stressing this is the default setting for all your workspaces could help here. WDYT?

```suggestion
                    <h4>Default Inactivity Timeout</h4>
```